### ISSUE TITLE
Buttons do not contain correct Aria code or alt text when buttons are linked

### ISSUE THREAD

**Issue by tomtolkien** [tomusborne/generateblocks#487]
### Description
The buttons are failing with: 'Aria Labels', and 'Links do not have a discernable name' when tested with Google Light House.

When Aria Labels are set in the button options, Lighthouse throws an error "[aria-*] attributes do not match their roles". 

When buttons are used for social SVG icon links, there no way to attach alt text to the button. Light House then throws a "no dscernable name" error. 

### Steps to reproduce
1. 
Creatre a social button (SVG) using Generateblocks. 
2.
Create a url link from the button to a social network.
3.
Test with Lighthouse. 

Easy way to replicate:
Use Generate Blocks Site Library - select "Social Media Icons -8" - publish page/post - inspect lighthouse results under "Accessibility" and issues will be apparent.

#### Expected behavior
Button should have correct ARIA label. 
URL button should have alt text. 

#### Actual behavior
Error on ARIA label.
No option to include ALT text. 

#### References: 
...

-----

**Comment by tomusborne** [comment]
Hey @tomtolkien - thanks for the report!

Just to confirm, this only seems like an issue when the Button has no link. Can you confirm?

If you're using 1.5, you may want to try this branch as there was a bug where the `aria-label` was being added to the icon element as well by mistake: https://github.com/tomusborne/generateblocks/pull/493

**Comment by tomusborne** [comment]
This shouldn't be an issue if:

1. You're using 1.4.3 or this branch: https://github.com/tomusborne/generateblocks/tree/release/1.5.0-beta.2
2. Your buttons have URLs set

**Comment by tomusborne** [comment]
If there is no text in the link, yes. Once you add an `aria-label`, that issue should go away.

**Comment by tomusborne** [comment]
Going to close this as I don't believe it's an issue anymore (1.5-beta.2) as long as a URL is set for the button.

Feel free to re-open if needed.

**Comment by tomtolkien** [comment]
I've checked again using 1.5-beta.2, but I'm still seeing the following in Lighthouse. 

Icon with link - '[aria-*] attributes do not match their roles'

Icon without link - 'Links do not have a discernible name'

The only scenario that passes Lighthouse is an icon with text, which is not really a scenario anyone is going to use?
